programs/loadsheet/loadsheet.py

```diff
@@ -120,8 +120,11 @@ def __init__(
 		# 			*[_REQ_INPUT_HEADERS+_REQ_OUTPUT_HEADERS if has_normalized_fields
 		# 			else _REQ_INPUT_HEADERS]))
 		# # end by sypks
```

Comment:
And add comment here explaining what this for loop is doing.

Reply:
done